karma.conf.js

```diff
@@ -27,23 +27,43 @@ module.exports = function(config) {
 
       'jujugui/static/gui/build/app/assets/javascripts/handlebars.runtime.js',
 
-      'jujugui/static/gui/build/app/components/**/*.js',
-      'jujugui/static/gui/build/**/test-*.js'
+      'jujugui/static/gui/src/app/components/**/*.hbs',
+      'jujugui/static/gui/src/app/components/**/*.js'
```

Comment:
With the preprocessors setup, Karma is smart enough to know that the source files (the files to be watched) are here, but the output files (the files to be served) are elsewhere.